diff --git a/Emby.Server.Implementations/Library/LiveStreamHelper.cs b/Emby.Server.Implementations/Library/LiveStreamHelper.cs
index 20624cc7a..f82c74d58 100644
--- a/Emby.Server.Implementations/Library/LiveStreamHelper.cs
+++ b/Emby.Server.Implementations/Library/LiveStreamHelper.cs
@@ -148,19 +148,19 @@ namespace Emby.Server.Implementations.Library
 
                     if (width >= 3000)
                     {
-                        videoStream.BitRate = 30000000;
+                        videoStream.BitRate = 2500000;
                     }
                     else if (width >= 1900)
                     {
-                        videoStream.BitRate = 20000000;
+                        videoStream.BitRate = 1000001;
                     }
                     else if (width >= 1200)
                     {
-                        videoStream.BitRate = 8000000;
+                        videoStream.BitRate = 750000;
                     }
                     else if (width >= 700)
                     {
-                        videoStream.BitRate = 2000000;
+                        videoStream.BitRate = 420001;
                     }
                 }
 
